Article/Article.js

```diff
@@ -3,24 +3,35 @@
 class Article {
   constructor(element) {
     // assign this.element to the passed in article element
-    this.element;
+    this.element = element;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = element.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = "expand";
     // Set a click handler on the expandButton reference (or article element), calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle(event));
+    //this.expandedButton = element.querySelector('.expandButton');
+    //this.expandedButton.addEventListener('click',() => this.closedButton(event))
   }
+    
 
-  expandArticle() {
+  expandArticle(event) {
+    this.element.classList.toggle('article');
```

Comment:
you want this to be toggling 'article-open' instead of article, take a look at article.less to see why we want to toggle that css class and not 'article'